TAA/TAA.md

```diff
@@ -0,0 +1,141 @@
+                .--.     .:`   --   `:.     .--.         
+                    .:////`    --    `////:.             
+                    //////-  `.::.`  -//////             
+                    .////:` `//////` `:////.             
+                      ```   `//////`   ```               
+                             `-::-`                      
+                                                                    
+ ssSSSSSss                                         II               
+sS"     "Ss                                        ""               
+SS,                                                                 
+`SSsssss,     ,ooOOOoo,   Vv       vV  Rr,rRRRrr,  II  Nn,nNNNnn,   
+  `"""""Ss,  oO"     "Oo  `Vv     vV'  RRR'   "RR  II  NNN'   `"Nn  
+        `Ss  OO       OO   `Vv   vv'   RR          II  NN       NN  
+SSs     SSS  "Oo,   ,oO"    `Vv,vV'    RR          II  NN       NN  
+ "SSSSSSS"    `"OoooO"'       "V"      RR          II  NN       NN  
+```
+[https://sovrin.org/](https://sovrin.org/)
+## **Summary:**
+* This summary is provided to help you understand your obligations when writing to the Sovrin Ledger Networks-it does not have any legal effect or replace the full legal text of the agreement provided below it.
+    * This agreement grants you permission to write data to the Sovrin Ledger Networks under certain terms and conditions.
+    * You represent and warrant that the data you are writing does not violate any applicable laws or infringe the rights of any other party.
+    * You understand the data you are writing is public and permanent and there can be no guarantee of erasure. This includes public keys and payment addresses.
+    * If it is determined that the data you wrote violated this agreement, the operators of the network can take steps to block it from public access.
+    * The Sovrin Foundation makes no promises about the reliability or correctness of the data being stored on the Sovrin Ledger Networks or the operation of the Sovrin Ledger Networks.
+~--~
+## **Agreement: Approved by the Sovrin Board of Trustees 04 December 2019**
+This Transaction Author Agreement (the "**Agreement**") is entered into on the date you accepted this Agreement (the "**Effective Date**") between the Sovrin Foundation, a nonprofit corporation organized under the laws of the State of Utah, United States of America ("**Sovrin Foundation**"), and you ("**Transaction** ​ **Author**"), either an entity or a natural person acting as an Individual. Sovrin Foundation and Transaction Author are individually referred to herein as a  "Party" and collectively as the  "Parties". All references to "you" throughout this Agreement will include that person or entity. You represent that you are authorized to accept this Agreement on that person’s or entity’s behalf, and in the event you or the person or entity violates this Agreement, the person or entity agrees to be responsible to the Sovrin Foundation.
```

Comment:
> `("**Transaction** ​ **Author**")`

A symbol missing here?

Reply:
no missing symbol. there is a space and separate bolding of `transaction` and `author` as an artifact of the converter used. `("**Transaction Author**")` is a good improvement.